sleap/gui/dialogs/formbuilder.py

```diff
@@ -437,13 +460,22 @@ def add_item(self, item: Dict[Text, Any]):
 
         # double: show spinbox (number w/ up/down controls)
         elif item["type"] == "double":
-            field = QtWidgets.QDoubleSpinBox()
+            
+            #If the label of the item is Loss Weight then we will set the min and max to 0 and 1000
```

Comment:
[Google style block comments](https://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments) (also looks a bit long for a single line).